code/modules/mob/living/carbon/human/species_types/kepori.dm

```diff
@@ -41,6 +41,9 @@
 	/// See: [/datum/component/tackler/var/skill_mod]
 	var/skill_mod = 2
 
+	// Does this species have unique robotic limbs? (currently used in: kepori and vox)
```

Comment:
why are you adding this comment to everything

Reply:
Clarity more than anything; as well as it being mostly a leftover from the old PR. I don't see an issue with it sticking around as it is only a code comment

Reply:
its fine to be on the base type, but it doesnt need to exist on the subtypes
if anyone wants to know what it is they can just click on it with vscode and be linked to the original variable on /datum/species